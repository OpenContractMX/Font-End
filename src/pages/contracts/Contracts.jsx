import React from "react";
import { FilterPerTime } from "../../components/filter-per-time";
import { NavOfContracts } from "../../components/nav-of-contracts";
import { SectionContracts } from "../../components/section-contracts";

import "./Contracts.scss";

export const Contracts = () => {
  return (
    <main className="container-contracts">
      <NavOfContracts />
      <FilterPerTime />
      <SectionContracts />
    </main>
  );
};
