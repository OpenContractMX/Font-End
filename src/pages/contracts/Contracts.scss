@import "../../assets/scss/reset";
@import "../../assets/scss/variables";
@import "../../assets/scss/mixins";

.container-contracts {
  margin: 0 auto;
  width: 95%;
  display: flex;
  flex-direction: column;
  align-items: center;

  &__text {
    color: $color-dark-primary;
    @include subTittleMobile;
    margin-top: 32px;

    &--category,
    &--year {
      font-weight: bold;
    }
  }
}

@media (min-width: 768px) {
  .container-contracts {
    &__text {
      @include subTitleTablet;
    }
  }
}
