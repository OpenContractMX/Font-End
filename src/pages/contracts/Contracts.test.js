import React from "react";
import { BrowserRouter } from "react-router-dom";

import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import { Contracts } from "./Contracts";

it("[render content `statistic / Contracts`]", () => {
  const { asFragment } = render(
    <BrowserRouter>
      <Contracts />
    </BrowserRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
