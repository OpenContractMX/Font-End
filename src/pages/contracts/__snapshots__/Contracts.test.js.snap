// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[render content \`statistic / Contracts\`] 1`] = `
<DocumentFragment>
  <main
    class="container-contracts"
  >
    <nav
      class="nav-contracts"
    >
      <a
        href="/statistic"
      >
        <button
          class="nav-contracts--btn-volver"
          type="button"
        >
          <i
            class="fas fa-arrow-left icon-volver"
          />
          Volver
        </button>
      </a>
    </nav>
    <section
      class="filter-time"
    >
      <h2
        class="filter-time--title"
      >
        Filtrar por:
      </h2>
      <div
        class="filter-time__forms"
      >
        <form
          class="filter-time__forms-month"
          for="form-filter-month"
        >
          <label>
            Mes:
          </label>
          <select
            class="filter-time__forms-month--select"
          >
            <option
              value="Mes"
            >
              Mes
            </option>
            <option
              value="1"
            >
              Enero
            </option>
            <option
              value="2"
            >
              Febrero
            </option>
            <option
              value="3"
            >
              Marzo
            </option>
            <option
              value="4"
            >
              Abril
            </option>
            <option
              value="5"
            >
              Mayo
            </option>
            <option
              value="6"
            >
              Junio
            </option>
            <option
              value="7"
            >
              Julio
            </option>
            <option
              value="8"
            >
              Agosto
            </option>
            <option
              value="9"
            >
              Septiembre
            </option>
            <option
              value="10"
            >
              Octubre
            </option>
            <option
              value="11"
            >
              Noviembre
            </option>
            <option
              value="12"
            >
              Diciembre
            </option>
          </select>
        </form>
        <span>
          Ã“
        </span>
        <form
          class="filter-time__forms-quarter"
          for="form-filter-quarter"
        >
          <label>
            Trimestre:
          </label>
          <select
            class="filter-time__forms-quarter--select"
          >
            <option
              value="Trimestre"
            >
              Trimestre
            </option>
            <option
              value="1"
            >
              Primer Trimestre
            </option>
            <option
              value="2"
            >
              Segundo Trimestre
            </option>
            <option
              value="3"
            >
              Tercer Trimestre
            </option>
            <option
              value="4"
            >
              Cuarto Trimestre
            </option>
          </select>
        </form>
      </div>
    </section>
    <section
      class="section-contracts"
    >
      <p
        class="section-contracts--filter"
      >
        Haz un filtro por mes o trimestre, para traer los datos
      </p>
    </section>
  </main>
</DocumentFragment>
`;
